module/testmodule/main.go

```diff
@@ -32,6 +32,9 @@ func main() {
 }
 
 func mainWithArgs(ctx context.Context, args []string, logger logging.Logger) error {
+	if os.Getenv("VIAM_TESTMODULE_PANIC") != "" {
+		panic("there is no flavor town")
```

Comment:
😆 is this a reference to something?

Reply:
hah, forgot i used this... I imagine this is a valid reason to panic for some people